woocommerce/wp-e2e-page-objects
# wp-e2e-webdriver package update

### Description

`wp-e2e-page-objects` was using a outdated version of `wp-e2e-webdriver` that depends on a outdated version of `chromedriver`. 
This PR updates the `wp-e2e-webdriver` package to the latest version to update the `chromedriver` dependency.

### Testing instructions

- Checkout this branch in your local environment
- Run `npm test`and make sure tests work.
